plugins/routed-eni/cni.go

```diff
@@ -67,6 +69,9 @@ type NetConf struct {
 	// veth device name. It should be no more than four characters, and
 	// defaults to 'eni'.
 	VethPrefix string `json:"vethPrefix"`
+
+	// MTU for eth0
+	Mtu string `json:"mtu"`
```

Comment:
Something I noticed... but golang naming practices generally encourage acronyms to be capitalized when exported, so this really should be "MTU", not "Mtu", like "CNIVersion" is capitalized above...

See more here: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I agree, should be all upper case.